src/apps/experimental/components/library/GenresSectionContainer.tsx

```diff
@@ -70,8 +70,8 @@ const GenresSectionContainer: FC<GenresSectionContainerProps> = ({
             centerText: true,
             cardLayout: false,
             shape: itemType === BaseItemKind.MusicAlbum ? 'overflowSquare' : 'overflowPortrait',
-            showParentTitle: itemType === BaseItemKind.MusicAlbum ? true : false,
-            showYear: itemType === BaseItemKind.MusicAlbum ? false : true
+            showParentTitle: (itemType === BaseItemKind.MusicAlbum),
+            showYear: (itemType !== BaseItemKind.MusicAlbum)
```

Comment:
The parentheses really aren't necessary

```suggestion
            showParentTitle: itemType === BaseItemKind.MusicAlbum,
            showYear: itemType !== BaseItemKind.MusicAlbum
```